RELEASE_NOTES.md

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-alpha.4
+--------------
+
+### New features
+
+- Add a C++ packaging process to build a cmake package from C++ traits
+  generated using `openasset-traitgen`.
+  [#24](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/issues/24)
+
```

Comment:
[fix] Here, and below - can we keep our double new line between versions convention (which is for readability), unless we decide globally to change this?

Reply:
We seem to be in a middle state here, the [pattern for the changelog](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/contributing/CHANGES.md#formatting) is single new line aside from between versions for readability. (We surveyed the world and there was a strong vote for readability in raw unrendered markup).

We can totally change this if desired, but should do everything everywhere all at once™.

Reply:
[04e9abe](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/25/commits/04e9abeabd786ec3e6e24ad087d906c8b6951431)

Consider me driven a bit whacky. Disabled yet another linter which had an opinion on markdown files, and I think it's as it should be now, single line spaced, except for between sections which is double spaced.

Reply:
Awesomesauce - thanks mate!